packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2825,7 +2825,7 @@ To ensure a successful password recovery lookup if an email address is associate
 
 ```json
 {
-  "resetPasswordUrl": "https://${yourOktaDomain}/reset_password/XE6wE17zmphl3KqAPFxO"
```

Comment:
Can you fix the other instances of this URL on this page as well?

Reply:
Done, I missed the other reference 👍